pkgs/development/python-modules/scikits-odes/default.nix

```diff
@@ -18,6 +19,8 @@ buildPythonPackage rec {
   pname = "scikits.odes";
   version = "2.6.1";
 
+  disabledIf = isPy2;
```

Comment:
I'd put this in a separate commit - to make it clearer for scikits' maintainers, or at least note this change at the same commit, in a new line.

Reply:
It should be: `disabled = isPy27`.

Reply:
Test the build with `nix-build -A python27Packages.scikits-odes`.

Reply:
> It should be: `disabled = isPy27`.

I find it strange since isPy2 is defined as well https://github.com/NixOS/nixpkgs/blob/c6975c45c3d65bd70c6566d1dcd0604eeba1cdc4/pkgs/development/interpreters/python/default.nix#L31

Reply:
I think it is part of the python's interpreter's attributes, but it is not available in the scope of `python-modules.nix`'s `callPackage` function.